src/libUtils/UpgradeManager.cpp

```diff
@@ -499,12 +514,14 @@ bool UpgradeManager::LoadInitialDS(vector<PubKey>& initialDSCommittee) {
       vector<std::string> tempDsComm_string{ReadDSCommFromFile()};
       initialDSCommittee.clear();
       for (auto ds_string : tempDsComm_string) {
-        initialDSCommittee.push_back(
-            PubKey(DataConversion::HexStrToUint8Vec(ds_string), 0));
+        bytes pubkeyBytes;
```

Comment:
As most of the inputs here are read from the file (and so it can be possible for the file to be the source of error), maybe we need to add warning log messages wherever the dataconversion fails in this function